packages/fxa-content-server/app/tests/spec/views/get_flow.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable camelcase */
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import $ from 'jquery';
+import Account from 'models/account';
+import { assert } from 'chai';
+import sinon from 'sinon';
+import User from 'models/user';
+import View from 'views/get_flow';
+import Notifier from 'lib/channels/notifier';
+import WindowMock from '../../mocks/window';
+import Url from '../../../scripts/lib/url';
+
+describe('views/get_flow', function() {
+  let account;
+  let user;
+  let view;
+
+  const flowData = {
+    deviceId: 'foo',
+    flowBeginTime: 987654,
+    flowId: 'h2k3v5',
+  };
+
+  function render() {
+    return view.render().then(() => view.afterVisible());
+  }
+
+  beforeEach(function() {
+    account = new Account();
+    user = new User();
+
+    view = new View({
+      user,
+      notifier: new Notifier(),
+      window: new WindowMock(),
+    });
+
+    sinon.stub(user, 'sessionStatus').callsFake(() => Promise.resolve(account));
+    sinon.stub(view, 'getSignedInAccount').callsFake(() => account);
```

Comment:
This is probably copied from existing tests.  Just FYI, `sinon.stub().resolves()` and `sinon.stub().returns()` should work here.